.github/workflows/oss.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - if: github.event.pull_request != null && steps.changes.outputs.ui == 'true'
         run: echo "PROJECT=171" >> "$GITHUB_ENV"
 
-      - uses: actions/add-to-project@v0.3.0    # TSCCR: no entry for repository "actions/add-to-project"
+      - uses: actions/add-to-project@a9f041ddd462ed185893ea1024cec954f50dbe42 # v0.3.0 # TSCCR: no entry for repository "actions/add-to-project"
```

Comment:
Just double-checked other instances of this message. Do we want to include the version stamp comment in this? I don't see any harm, but just something that jumped out at me.

Reply:
I think it's ok, since the TSCCR comment is still present